src/applications/hca/components/PreSubmitNotice/index.jsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { VaCheckbox } from '@department-of-veterans-affairs/component-library/dist/react-bindings';
+
+const PreSubmitNotice = props => {
+  const { formData, preSubmitInfo, showError, onSectionComplete } = props;
+  const { field, required } = preSubmitInfo;
+  return (
+    <>
+      <h3>Agreement</h3>
+      <p>By submitting this application, you agree to these statements:</p>
+      <ul data-testid="hca-agreement-statements">
+        <li>
+          You’ll pay any VA copays for care or services (including urgent care)
+          that may apply, based on your priority group and other factors.
+        </li>
+        <li>
+          You agree that we can contact you at the email, home phone number, and
+          mobile phone number you gave us in the application.
+        </li>
+        <li>
+          You agree to the assignment of benefits so we can bill your other
+          health insurance or other responsible party for charges of
+          nonservice-connected VA medical care or services.
+        </li>
+        <li>
+          You’ve read and accept our privacy policy.
+          <br />
```

Comment:
Not sure if this is happening here, but Josh pointed this [problem with `br`s to me before](https://github.com/nvaccess/nvda/issues/6400)